fuse/mount_darwin.go

```diff
@@ -14,33 +14,106 @@ import (
 	"unsafe"
 )
 
+const FUSET_SRV_PATH = "/usr/local/bin/go-nfsv4"
```

Comment:
Shouldn't this be ` /Library/Application Support/fuse-t/....` acc. to  https://github.com/macos-fuse-t/fuse-t/wiki ?